lib/shared/loading.dart

```diff
@@ -0,0 +1,18 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_spinkit/flutter_spinkit.dart';
+
+class Loading extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
You can just use a [circular progress indicator](https://api.flutter.dev/flutter/material/CircularProgressIndicator-class.html). It will look a bit out of place compared to the rest of the design.

Reply:
Yes, but I can also use this one and it could be edited a bit to fit the design. Different loaders from this package look cool, much more interesting than the default circular one. Ezy + respect points.